conf/cmi/core.entity_view_display.paragraph.hero.default.yml

```diff
@@ -72,15 +72,6 @@ content:
     third_party_settings: {  }
     weight: 1
     region: content
-  field_display_search_field:
```

Comment:
This configuration needs to be checked and if the field should be visible, the rewrite needs to be added to `public/modules/custom/kaupunkitieto_config/config/rewrite`.